src/decoder.rs

```diff
@@ -901,7 +900,7 @@ impl Decoder {
             Format::PeekNot(a) => {
                 const MAX_LOOKAHEAD: usize = 1024;
                 match a.match_bounds(compiler.module).max {
-                    None => return Err(format!("PeekNot cannot require unbounded lookahead")),
+                    None => return Err("PeekNot cannot require unbounded lookahead".to_string()),
```

Comment:
I'd rather stick with format! here as it's shorter and easier to add args when debugging